worker.py

```
@@ -0,0 +1,193 @@
+            body=email.body,
+            is_html=bool(email.is_html)
+        )
+        
+        # Update status to sent
+        await email_service.update_status(db, email_id, EmailStatus.SENT)
+        await queue_service.complete(email_id)
+        
+        logger.info(f"Successfully sent email {email_id}")
+        return True
+        
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)